docs/cron-jobs.md

```diff
@@ -10,7 +10,7 @@ import TabItem from '@theme/TabItem';
 
 ## Create CRON Job
 
```

Comment:
@nnascim Here we also need to allow the cron expression variable in the less.config file. maybe we should mention it in this cron section

Reply:
@savylopes it's solved on line 37.